priv/scripts/otp/otp.sh

```diff
@@ -5,12 +5,21 @@ set -euox pipefail
 ref_name=$1
 ref=$2
 linux=$3
+arch=$4
 
 source ${SCRIPT_DIR}/utils.sh
 
-echo "Building $1 $2 $3"
+echo "Building $1 $2 $3 $4"
+
+if [[ "$arch" == "arm64" ]]; then
+  container="otp-build-${linux}-${ref_name}-arm64"
+elif [[ "$arch" == "amd64" ]]; then
+  container="otp-build-${linux}-${ref_name}-amd64"
+else
+  echo "Invalid architecture. Please specify either 'arm64' or 'amd64'."
+  exit 1
+fi
```

Comment:
Different container names are not needed as the builds will run on different machines.

```suggestion
```